### PR TITLE
Update UI with wrap text, logo

### LINKED ISSUE #74: Update UI with wrap text, logo
Update UI to include wrap text functionality for the command line, as well as a new logo

### PR DESCRIPTION
**Pull request by colinhia**
Updated the AgentAssist app with a new logo, and updated the command line box to use a TextArea instead of a TextField, allowing for scrolling down the command line

Closes #74

### CODE REVIEW

File: src/main/java/seedu/address/ui/CommandBox.java [hunk 1]
```diff
@@ -53,19 +57,17 @@ private void handleCommandEntered() {
...
     }
 
     /**
      * Sets the command box style to indicate a failed command.
      */
     private void setStyleToIndicateCommandFailure() {
-        ObservableList<String> styleClass = commandTextField.getStyleClass();
-
+        ObservableList<String> styleClass = commandTextArea.getStyleClass();
         if (styleClass.contains(ERROR_STYLE_CLASS)) {
```
Comment (iamdiluxedbutcooler) [inline src/main/java/seedu/address/ui/CommandBox.java:68]: ERROR_STYLE_CLASS might need to be adjusted for a TextArea, not sure but worth for double checking!

File: src/main/resources/images/agent_assist_logo.png
Comment (iamdiluxedbutcooler) [inline src/main/resources/images/agent_assist_logo.png:1]: Good profile pic, looks good on UI
Screenshot 2567-10-17 at 02 33 49

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
          title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
```
Comment (iamdiluxedbutcooler) [inline src/main/resources/view/MainWindow.fxml:15]: The title of the app is still Address App. Consider changing to our app name "Agent Assist"!

**Review by iamdiluxedbutcooler — APPROVED** [review]
Overall LGTM! I'd set as approve first but might need a minor change on the app title. Thanks for your hardworking on JavaFX UI!

File: src/main/resources/view/MainWindow.fxml (hunk 2 above)
Comment (ZShunRen) [reply src/main/resources/view/MainWindow.fxml:15]: Will be changed by my separate PR, see #83

File: src/main/java/seedu/address/ui/CommandBox.java (hunk 1 above)
Comment (ZShunRen) [reply src/main/java/seedu/address/ui/CommandBox.java:68]: The text itself is affected by the ERROR_STYLE_CLASS, so LGTM

**Review by ZShunRen — APPROVED** [review]
LGTM

**Pull request merged**